src/Components/MovieDetails/Header/Header.js

```diff
@@ -1,56 +1,62 @@
-import React from "react";
+import React, { useState, useContext, useEffect } from "react";
+import { useTranslation } from "react-i18next";
 import { useParams, useHistory } from "react-router-dom";
-import "./Header.scss";
 
-import { ModalUpdaterContext } from "../../Context/ModalContext.js";
-import MovieDetails from "../MovieDetails/MovieDetails.js";
+import MovieDetails from "../MovieDetails.js";
 import Search from "./Search.js";
+import { FILMS_PATH } from "../../../Core/Constants";
+import { I18N_KEYS } from "../../../Core/I18N";
+import { ModalUpdaterContext } from "../../../HOCs/Context/ModalContext.js";
+
+import "./Header.scss";
 
 export default function Header() {
+    const { t } = useTranslation();
     const { id } = useParams();
     const history = useHistory();
-    const setModalOptions = React.useContext(ModalUpdaterContext);
-    const [isOpen, setIsOpen] = React.useState(false);
+    const setModalOptions = useContext(ModalUpdaterContext);
+    const [isOpen, setIsOpen] = useState(false);
 
-    React.useEffect(() => {
-        let path = history.location.pathname.split('/')[1]
-        if(path === 'films') {
-            console.log(path)
+    useEffect(() => {
+        let path = history.location.pathname.split("/")[1];
```

Comment:
"path" in this context doesn't seem to change, so it's more suitable to use "const" instead of "let" here.
Apart from adding some semantic context ("const" means this variable doesn't get a different value in the current context), this approach also (speculatively) improves overall performance.